dateparser/conf.py

```diff
@@ -49,7 +49,7 @@ def get_key(cls, settings=None):
             return "default"
 
         keys = sorted(["%s-%s" % (key, str(settings[key])) for key in settings])
-        return hashlib.md5("".join(keys).encode("utf-8")).hexdigest()
+        return hashlib.new("md5", "".join(keys).encode("utf-8"), usedforsecurity=False).hexdigest()
```

Comment:
Why switch to  `new`? 

This seems also possible:

```suggestion
        return hashlib.md5("".join(keys).encode("utf-8"), usedforsecurity=False).hexdigest()
```